feature/xhr/aggregate/index.js

```diff
@@ -168,3 +194,17 @@ function onEventsHarvestFinished(result) {
     sentAjaxEvents = []
   }
 }
+
+function exceedsSizeLimit(payload, maxPayloadSize) {
+  maxPayloadSize = maxPayloadSize || MAX_PAYLOAD_SIZE
+  return window.Blob ? new Blob([payload]).size > maxPayloadSize : false
+}
+
+function chunk(arr, chunkSize) {
+  if (chunkSize <= 0) throw new Error('Invalid chunk size')
```

Comment:
Can this happen? If yes, we should handle the error, so that the error does not show up as uncaught error in the customer app (which the customer cannot address).

Reply:
fair, i'll remove this as I don't think this will be possible the way it is written.  This was mostly a developer safety rail so Its not needed and I don't want errors to propagate to NR1